fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatService.java

```diff
@@ -99,7 +99,7 @@ long syncServiceAccount(
    * @return ignored.
    */
   @POST("/roles/{userId}")
-  Response loginUser(
+  Call<Void> loginUser(
       @Path("userId") String userId, @Body String ignored /* retrofit requires this */);
```

Comment:
does retrofit2 still require a body param?

Reply:
It was introduced to support gate's fiat call but looks like it was not needed any more even with retrofit1.  
Ref - https://github.com/spinnaker/gate/pull/1737 and [this](https://github.com/spinnaker/gate/blob/master/gate-core/src/main/java/com/netflix/spinnaker/gate/services/PermissionService.java#L81-L82)

Reply:
I will remove it

Reply:
Removed it. Will raise the Gate PR after this PR is merged.

Reply:
I'd love to see the draft gate PR before we merge this one.